config/default.yml

```diff
@@ -63,96 +64,133 @@ Metrics/ModuleLength:
+# https://docs.rubocop.org/rubocop/cops_layout.html#layoutmultilineoperationindentation
+Layout/MultilineOperationIndentation:
+  Description: Checks indentation of binary operations that span more than one line.
+  EnforcedStyle: indented
+
 #### Hash
+# https://docs.rubocop.org/rubocop/cops_layout.html#layouthashalignment
 Layout/HashAlignment:
   EnforcedColonStyle: table
   EnforcedHashRocketStyle: table
 
+# https://docs.rubocop.org/rubocop/cops_style.html#stylehashsyntax
 Style/HashSyntax:
   EnforcedStyle: ruby19_no_mixed_keys
 
+# https://docs.rubocop.org/rubocop/cops_style.html#stylehashtransformkeys
 Style/HashTransformKeys:
   Enabled: false
 
+# https://docs.rubocop.org/rubocop/cops_style.html#stylehashtransformvalues
 Style/HashTransformValues:
   Enabled: false
 
-Layout/MultilineOperationIndentation:
-  Description: Checks indentation of binary operations that span more than one line.
-  EnforcedStyle: indented
-
 #### Rest
+# https://docs.rubocop.org/rubocop/cops_style.html#styledocumentation
 Style/Documentation:
   Enabled: false
 
+# https://docs.rubocop.org/rubocop/cops_style.html#styledoublenegation
 Style/DoubleNegation:
   Enabled: false
 
+# https://docs.rubocop.org/rubocop/cops_style.html#styleexponentialnotation
 Style/ExponentialNotation:
-  # https://docs.rubocop.org/rubocop/cops_style.html#styleexponentialnotation
   EnforcedStyle: engineering
 
+# https://docs.rubocop.org/rubocop/cops_style.html#stylenumericliterals
 Style/NumericLiterals:
   Description: Add underscores to large numeric literals to improve their readability.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#underscores-in-numerics
   Enabled: false
 
+# https://docs.rubocop.org/rubocop/cops_style.html#styleperlbackrefs
 Style/PerlBackrefs:
   Enabled: false
 
+# https://docs.rubocop.org/rubocop/cops_style.html#styleraiseargs
 Style/RaiseArgs:
   EnforcedStyle: compact
 
+# https://docs.rubocop.org/rubocop/cops_style.html#styleregexpliteral
 Style/RegexpLiteral:
   Enabled: false
 
+# https://docs.rubocop.org/rubocop/cops_style.html#stylesignalexception
 Style/SignalException:
   EnforcedStyle: only_raise
 
+# https://docs.rubocop.org/rubocop/cops_style.html#stylestringliterals
 Style/StringLiterals:
   EnforcedStyle: double_quotes
 
+# https://docs.rubocop.org/rubocop/cops_style.html#styleoptionalbooleanparameter
 Style/OptionalBooleanParameter:
   Enabled: false
 
-
 ### RSpec
 
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecdescribeclass
 RSpec/DescribeClass:
   Exclude:
     - spec/integration/**/*
 
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecexamplelength
 RSpec/ExampleLength:
   Enabled: false
 
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecexamplewording
 RSpec/ExampleWording:
   IgnoredWords:
     - only
   CustomTransform:
     have: has
 
-RSpec/FactoryBot/SyntaxMethods:
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecspecfilepathformat
+RSpec/SpecFilePathFormat:
```

Comment:
Enabling it and ignoring `spec/integrations` here could also work.
What do you think?

Reply:
disabling sounds fine to me